Aptoide/bds-sdk
# Intent being returned in the getBuyIntent when wallet is not installed

**What does this PR do?**

   This PR returns an Intent with an activity rather than a response code on the getBuyIntent 
   method when the wallet is not installed. This way, even if the current activity is closed, the pop up 
   appears when the launch billing flow is called.
 

**Database changed?**

   No

**Where should the reviewer start?**

- AppcoinsBillingStubHelper.java
- InstallDialogActivity.java
-  WalletUtils.java

**How should this be manually tested?**

This PR should be tested in Prod with no Wallet installed:

1- Skip the poa WalletInstall popup

2-Press the LaunchBillingFlow button and wait for the popup to come up

3-Press Skip to confirm that the activity is finished and both the popup and the new layout disappear and then press  the LaunchBillingFlow button again and see if the install button from the popup redirects you to the wallet's store page (GooglePlay or Aptoide)

**What are the relevant tickets?**

  APPC-1202

https://aptoide.atlassian.net/secure/RapidBoard.jspa?rapidView=52&projectKey=APPC&modal=detail&selectedIssue=APPC-1202&quickFilter=288

**Questions:**

Can we make the layout from the new activity disappear faster when the Skip button is pressed?

**Code Review Checklist**
